rest/model/versions.go

```diff
@@ -0,0 +1,30 @@
+package model
+
+import "errors"
+
+type APIVersions struct {
+	// whether or not the version element actually consists of multiple inactive
+	// versions rolled up into one
+	RolledUp bool `json:"rolled_up"`
+
+	Versions []APIVersion `json:"versions"`
+}
+
+type BuildList struct {
+	BuildVariant string              `json:"build_variant"`
+	Builds       map[string]APIBuild `json:"builds"`
+}
+
+type VersionVariantData struct {
+	Rows          map[string]BuildList `json:"rows"`
+	Versions      []APIVersions        `json:"versions"`
+	BuildVariants []string             `json:"build_variants"`
+}
+
+func (v *VersionVariantData) BuildFromService(h interface{}) error {
+	return nil
```

Comment:
nit, shouldn't this also return a not implemented error?

Reply:
This is actually the implementation. I need to return this struct as an API model, but the model doesn't need to do anything on its own